src/aig-common/modules/iam/components/user-new/user-new.component.ts

```diff
@@ -25,9 +27,6 @@ export class AigUserNewComponent implements OnInit {
         private eventService: EventService,
     ) { }
 
-    private userNewForm: FormGroup;
-    private userDTO: UserDTO;
-
     ngOnInit(): void {
         this.userNewForm = this._formBuilder.group({
             firstName: ['', Validators.required],
```

Comment:
restore original of this file

Reply:
I don't modify this file

Reply:
if stay there is modified file, if you need original file you can fount it here: https://github.com/EneaSys/aig_console

Reply:
this is the oriinal:
[https://github.com/EneaSys/aig_console/blob/develop/src/aig-common/modules/iam/components/user-new/user-new.component.ts](url)